include/nanobind/eigen/sparse.h

```diff
@@ -0,0 +1,125 @@
+	    object matrix_type = module_::import_("scipy.sparse").attr(row_major ? "csr_matrix" : "csc_matrix");
+	    if (!obj.type().is(matrix_type)) {
+                obj = matrix_type(obj);
+	    }
+	} catch (const python_error &) {
+	    return false;
+	}
+
+	if (object data_o = obj.attr("data"); !scalar_caster.from_python(data_o, flags, cleanup))
+	    return false;
+        ScalarTensor& values = scalar_caster.value;
+
+	if (object indices_o = obj.attr("indices"); !storageindex_caster.from_python(indices_o, flags, cleanup))
+	    return false;
+        StorageIndexTensor& inner_indices = storageindex_caster.value;
+
+	if (object indptr_o = obj.attr("indptr"); !scalar_caster.from_python(indptr_o, flags, cleanup))
+	    return false;
+        StorageIndexTensor& outer_indices = storageindex_caster.value;
+
+	object shape_o = obj.attr("shape"), nnz_o = obj.attr("nnz");
+	Index rows, cols, nnz;
+	try {
+	    if (len(shape_o) != 2)
+		return false;
+	    rows = cast<Index>(shape_o[0]);
+	    cols = cast<Index>(shape_o[1]);
+	    nnz = cast<Index>(nnz_o);
+	} catch (const python_error &) {
+	    return false;
+	}
+
+        value = EigenMapSparseMatrix<Scalar,
+                                     T::Flags &(Eigen::RowMajor | Eigen::ColMajor),
+                                     StorageIndex>(std::move(rows),
+						   std::move(cols),
+                                                   std::move(nnz),
+                                                   outer_indices.data(),
+                                                   inner_indices.data(),
+                                                   values.data());
+
+        return true;
+    }
+
+    static handle from_cpp(T &&v, rv_policy policy, cleanup_list *cleanup) noexcept {
+        if (policy == rv_policy::automatic ||
+            policy == rv_policy::automatic_reference)
+            policy = rv_policy::move;
+
+        return from_cpp((const T &) v, policy, cleanup);
+    }
+
+    static handle from_cpp(const T &v, rv_policy policy, cleanup_list *cleanup) noexcept {
+	// FIXME: Looks like UB. Is there anything that can be done about this?
+	T & src = const_cast<T &>(v);
+	src.makeCompressed();
+
+	object matrix_type = module_::import_("scipy.sparse").attr(row_major ? "csr_matrix" : "csc_matrix");
+
+	// FIXME: These casts are ugly! Is there another way to write this?
+	const size_t data_shape[] = { static_cast<size_t>(v.nonZeros()) };
+	const size_t outerIndices_shape[] = { static_cast<size_t>((row_major ? v.rows() : v.cols()) + 1) };
+        ScalarTensor data(src.valuePtr(), 1, data_shape);
+        StorageIndexTensor outerIndices(src.outerIndexPtr(), 1, outerIndices_shape);
+        StorageIndexTensor innerIndices(src.innerIndexPtr(), 1, data_shape);
+
+        return matrix_type(make_tuple(
+                               std::move(data), std::move(innerIndices), std::move(outerIndices)),
+                           make_tuple(v.rows(), v.cols()))
+            .release();
+    }
```

Comment:
Currently the test segfaults in the return statement, but I have no idea where this comes from.

<details>

<summary>Backtrace</summary>

```cpp
Thread 1 (Thread 0x7ffff7891740 (LWP 143127) "python"):
#0  0x00007ffff791ebc7 in __pthread_kill_implementation () from /nix/store/9xfad3b5z4y00mzmk2wnn4900q0qmxns-glibc-2.35-224/lib/libc.so.6
No symbol table info available.
#1  0x00007ffff78d1b46 in raise () from /nix/store/9xfad3b5z4y00mzmk2wnn4900q0qmxns-glibc-2.35-224/lib/libc.so.6
No symbol table info available.
#2  0x00007ffff78bc4b5 in abort () from /nix/store/9xfad3b5z4y00mzmk2wnn4900q0qmxns-glibc-2.35-224/lib/libc.so.6
No symbol table info available.
#3  0x00007fffae4899fd in __gnu_cxx::__verbose_terminate_handler() [clone .cold] () from /home/henri/Code/nanobind/build/tests/test_eigen_ext.cpython-310-x86_64-linux-gnu.so
No symbol table info available.
#4  0x00007fffae4ffdca in __cxxabiv1::__terminate(void (*)()) () from /home/henri/Code/nanobind/build/tests/test_eigen_ext.cpython-310-x86_64-linux-gnu.so
No symbol table info available.
#5  0x00007fffae4ffe35 in std::terminate() () from /home/henri/Code/nanobind/build/tests/test_eigen_ext.cpython-310-x86_64-linux-gnu.so
No symbol table info available.
#6  0x00007fffae49633f in __clang_call_terminate () from /home/henri/Code/nanobind/build/tests/test_eigen_ext.cpython-310-x86_64-linux-gnu.so
No symbol table info available.
#7  0x00007fffae4b3878 in nanobind::detail::type_caster<Eigen::SparseMatrix<float, 1, int>, int>::from_cpp (v=..., policy=nanobind::rv_policy::move, cleanup=0x7fffffff2bb0) at /home/henri/Code/nanobind/include/nanobind/eigen/sparse.h:107
        src = @0x7fffffff29c8: {
          <Eigen::SparseCompressedBase<Eigen::SparseMatrix<float, 1, int> >> = {
            <Eigen::SparseMatrixBase<Eigen::SparseMatrix<float, 1, int> >> = {
              <Eigen::EigenBase<Eigen::SparseMatrix<float, 1, int> >> = {<No data fields>}, 
              members of Eigen::SparseMatrixBase<Eigen::SparseMatrix<float, 1, int> >:
              m_isRValue = false
            }, <No data fields>}, 
          members of Eigen::SparseMatrix<float, 1, int>:
          m_outerSize = 5,
          m_innerSize = 6,
          m_outerIndex = 0xabc890,
          m_innerNonZeros = 0x0,
          m_data = {
            m_values = 0xac7f30,
            m_indices = 0xacd210,
            m_size = 13,
            m_allocatedSize = 13
          }
        }
        matrix_type = {
          <nanobind::handle> = {
            <nanobind::detail::api<nanobind::handle>> = {
              <nanobind::detail::api_tag> = {<No data fields>}, <No data fields>}, 
            members of nanobind::handle:
            static Name = <optimized out>,
            m_ptr = 0xae74c0
          }, 
          members of nanobind::object:
          static Name = <optimized out>
        }
        data_shape = {13}
        outerIndices_shape = {6}
        data = {
          m_handle = 0x7fffae42b970,
          m_tensor = {
            data = 0xac7f30,
            device = {
              device_type = 1,
              device_id = 0
            },
            ndim = 1,
            dtype = {
              code = 2 '\002',
              bits = 32 ' ',
              lanes = 1
            },
            shape = 0x7fff6ba88150,
            strides = 0x7fff6ba88120,
            byte_offset = 0
          }
        }
        outerIndices = {
          m_handle = 0x7fffae42b790,
          m_tensor = {
            data = 0xabc890,
            device = {
              device_type = 1,
              device_id = 0
            },
            ndim = 1,
            dtype = {
              code = 0 '\000',
              bits = 32 ' ',
              lanes = 1
            },
            shape = 0x7fff6ba88140,
            strides = 0x7fff6ba88160,
            byte_offset = 0
          }
        }
        innerIndices = {
          m_handle = 0x7fffae42b870,
          m_tensor = {
            data = 0xacd210,
            device = {
              device_type = 1,
              device_id = 0
            },
            ndim = 1,
            dtype = {
              code = 0 '\000',
              bits = 32 ' ',
              lanes = 1
            },
            shape = 0x7fff6ba88170,
            strides = 0x7fff6ba88180,
            byte_offset = 0
          }
        }
#8  0x00007fffae4b31be in nanobind::detail::type_caster<Eigen::SparseMatrix<float, 1, int>, int>::from_cpp (v=..., policy=nanobind::rv_policy::move, cleanup=0x7fffffff2bb0) at /home/henri/Code/nanobind/include/nanobind/eigen/sparse.h:99
No locals.
#9  0x00007fffae494d9f in nanobind::detail::func_create<false, true, nanobind_init_test_eigen_ext(nanobind::module_&)::$_19, Eigen::SparseMatrix<float, 1, int>, , , nanobind::scope, nanobind::name>(nanobind_init_test_eigen_ext(nanobind::module_&)::$_19&&, Eigen::SparseMatrix<float, 1, int> (*)(), std::integer_sequence<unsigned long>, nanobind::scope const&, nanobind::name const&)::{lambda(void*, _object**, unsigned char*, nanobind::rv_policy, nanobind::detail::cleanup_list*)#1}::operator()(void*, _object**, unsigned char*, nanobind::rv_policy, nanobind::detail::cleanup_list*) const (this=0x0, p=0x7fffaee26468, args=0x7fffaee72538, args_flags=0x7fffffff2bf0 "\001\001\001\001\001\001\001\001", policy=nanobind::rv_policy::automatic, cleanup=0x7fffffff2bb0) at /home/henri/Code/nanobind/include/nanobind/nb_func.h:128
        cap = 0x7fffaee26468
        result = 0x7ffff660d3b0
#10 nanobind::detail::func_create<false, true, nanobind_init_test_eigen_ext(nanobind::module_&)::$_19, Eigen::SparseMatrix<float, 1, int>, , , nanobind::scope, nanobind::name>(nanobind_init_test_eigen_ext(nanobind::module_&)::$_19&&, Eigen::SparseMatrix<float, 1, int> (*)(), std::integer_sequence<unsigned long>, nanobind::scope const&, nanobind::name const&)::{lambda(void*, _object**, unsigned char*, nanobind::rv_policy, nanobind::detail::cleanup_list*)#1}::__invoke(void*, _object**, unsigned char*, nanobind::rv_policy, nanobind::detail::cleanup_list*) (p=0x7fffaee26468, args=0x7fffaee72538, args_flags=0x7fffffff2bf0 "\001\001\001\001\001\001\001\001", policy=nanobind::rv_policy::automatic, cleanup=0x7fffffff2bb0) at /home/henri/Code/nanobind/include/nanobind/nb_func.h:105
No locals.
#11 0x00007fffae4c4f3f in nanobind::detail::nb_func_vectorcall_simple (self=0x7fffaee26440, args_in=0x7fffaee72538, nargsf=9223372036854775808, kwargs_in=0x0) at /home/henri/Code/nanobind/src/nb_func.cpp:732
        f = 0x7fffaee26468
        e = @0x40d0f0: {
          <std::exception> = {<No data fields>}, 
          members of nanobind::python_error:
          m_type = 0x0,
          m_value = 0x439fb0,
          m_trace = 0x7fffaee723b0,
          m_what = 0x28 <error: Cannot access memory at address 0x28>
        }
        k = 0
        pass = 1
        args_flags = "\001\001\001\001\001\001\001\001"
        fr = 0x7fffaee26468
        count = 1
        nargs_in = 0
        is_method = false
        is_constructor = false
        self_flags = 0
        result = 0x0
        self_arg = 0x0
        cleanup = {
          static Small = 6,
          m_size = 1,
          m_capacity = 6,
          m_data = 0x7fffffff2bc0,
          m_local = {0x0, 0xac2aa7be639ae800, 0x40d0f0, 0x7fffaee723b0, 0x40d0f0, 0x7ffff66483b0}
        }
        error_handler = 0x0
        fail = false
        none_ptr = 0x7ffff7fafe30 <_Py_NoneStruct>
#12 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#13 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#14 0x00007ffff7c41169 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#15 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#16 0x00007ffff7c41169 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#17 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#18 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#19 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#20 0x00007ffff7cf7cd8 in method_vectorcall () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#21 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#22 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#23 0x00007ffff7d02736 in _PyObject_Call_Prepend () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#24 0x00007ffff7d43b49 in slot_tp_call () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#25 0x00007ffff7c95c84 in _PyObject_MakeTpCall () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#26 0x00007ffff7c47f69 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#27 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#28 0x00007ffff7c475b9 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#29 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#30 0x00007ffff7c41169 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#31 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#32 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#33 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#34 0x00007ffff7cf7cd8 in method_vectorcall () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#35 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#36 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#37 0x00007ffff7d02736 in _PyObject_Call_Prepend () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#38 0x00007ffff7d43b49 in slot_tp_call () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#39 0x00007ffff7ca7b39 in PyObject_Call () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#40 0x00007ffff7c41169 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#41 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#42 0x00007ffff7c4669f in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#43 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#44 0x00007ffff7cf7cd8 in method_vectorcall () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#45 0x00007ffff7c46344 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#46 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#47 0x00007ffff7c41169 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#48 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#49 0x00007ffff7c4669f in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#50 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#51 0x00007ffff7c46344 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#52 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#53 0x00007ffff7c41169 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#54 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#55 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#56 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#57 0x00007ffff7cf7cd8 in method_vectorcall () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#58 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#59 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#60 0x00007ffff7d02736 in _PyObject_Call_Prepend () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#61 0x00007ffff7d43b49 in slot_tp_call () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#62 0x00007ffff7c95c84 in _PyObject_MakeTpCall () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#63 0x00007ffff7c47f69 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#64 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#65 0x00007ffff7c41169 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#66 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#67 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#68 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#69 0x00007ffff7cf7cd8 in method_vectorcall () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#70 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#71 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#72 0x00007ffff7d02736 in _PyObject_Call_Prepend () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#73 0x00007ffff7d43b49 in slot_tp_call () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#74 0x00007ffff7c95c84 in _PyObject_MakeTpCall () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#75 0x00007ffff7c47f69 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#76 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#77 0x00007ffff7c4669f in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#78 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#79 0x00007ffff7c4669f in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#80 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#81 0x00007ffff7c41169 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#82 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#83 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#84 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#85 0x00007ffff7cf7cd8 in method_vectorcall () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#86 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#87 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#88 0x00007ffff7d02736 in _PyObject_Call_Prepend () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#89 0x00007ffff7d43b49 in slot_tp_call () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#90 0x00007ffff7c95c84 in _PyObject_MakeTpCall () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#91 0x00007ffff7c47f69 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#92 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#93 0x00007ffff7c4669f in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#94 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#95 0x00007ffff7c49cf2 in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#96 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#97 0x00007ffff7dc18e8 in PyEval_EvalCode () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#98 0x00007ffff7e590ed in builtin_exec () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#99 0x00007ffff7db69bf in cfunction_vectorcall_FASTCALL () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#100 0x00007ffff7c4669f in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#101 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#102 0x00007ffff7c4669f in _PyEval_EvalFrameDefault () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#103 0x00007ffff7dc127f in _PyEval_Vector () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#104 0x00007ffff7e4e5c7 in pymain_run_module () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#105 0x00007ffff7e59f0b in Py_RunMain () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#106 0x00007ffff7e5a535 in Py_BytesMain () from /nix/store/5axq6aw8j3vcs2m7gi440cwpcckl7ql9-python3-3.10.9/lib/libpython3.10.so.1.0
No symbol table info available.
#107 0x00007ffff78bd24e in __libc_start_call_main () from /nix/store/9xfad3b5z4y00mzmk2wnn4900q0qmxns-glibc-2.35-224/lib/libc.so.6
No symbol table info available.
#108 0x00007ffff78bd309 in __libc_start_main_impl () from /nix/store/9xfad3b5z4y00mzmk2wnn4900q0qmxns-glibc-2.35-224/lib/libc.so.6
No symbol table info available.
#109 0x0000000000401075 in _start ()
No symbol table info available.
```

</details>

Reply:
Ah no, wait. It looks like `module_::import_("scipy.sparse")` throws and causes terminate.